packages/frontend-api/src/Component/Price/MoneyFormatterHelper.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Component\Price;
+
+use CommerceGuys\Intl\Currency\CurrencyRepositoryInterface;
+use Shopsys\FrameworkBundle\Component\CurrencyFormatter\CurrencyFormatterFactory;
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade;
+
+class MoneyFormatterHelper
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade
+     */
+    protected $currencyFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\CurrencyFormatter\CurrencyFormatterFactory
+     */
+    protected $currencyFormatterFactory;
+
+    /**
+     * @var \CommerceGuys\Intl\Currency\CurrencyRepositoryInterface
+     */
+    protected $intlCurrencyRepository;
+
+    /**
+     * MoneyFormatterHelper constructor.
+     *
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade $currencyFacade
+     * @param \Shopsys\FrameworkBundle\Component\CurrencyFormatter\CurrencyFormatterFactory $currencyFormatterFactory
+     * @param \CommerceGuys\Intl\Currency\CurrencyRepositoryInterface $intlCurrencyRepository
+     */
+    public function __construct(
+        CurrencyFacade $currencyFacade,
+        CurrencyFormatterFactory $currencyFormatterFactory,
+        CurrencyRepositoryInterface $intlCurrencyRepository
+    ) {
+        $this->currencyFacade = $currencyFacade;
+        $this->currencyFormatterFactory = $currencyFormatterFactory;
+        $this->intlCurrencyRepository = $intlCurrencyRepository;
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Money\Money $money
+     * @param int $domainId
+     * @return \Shopsys\FrameworkBundle\Component\Money\Money
+     */
+    public function getMoneyFormattedAsDecimalWithFractionDigitsByCurrencySetting(Money $money, int $domainId): Money
+    {
+        $locale = 'en';
```

Comment:
You should get locale for first domain here

Reply:
No it should not.

That is why we need this method. We want to have formatted price with the decimal dot (not comma).